package.json

```diff
@@ -52,12 +52,14 @@
     "qrcode.vue": "^1.7.0",
     "rxjs": "^6.6.7",
     "rxjs-etc": "^10.6.2",
+    "swiper": "^5.4.5",
```

Comment:
Do we need both packages: `swiper` and `vue-awesome-swiper`?

Reply:
Yes, we had to use those specific version to make it work our current Vue version